internal/trie/node/branch_encode_test.go

```diff
@@ -0,0 +1,514 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package node
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_Branch_Encode(t *testing.T) {
```

Comment:
```suggestion
func TestBranchEncode(t *testing.T) {
```

Reply:
I prefer formatting my test names with underscores as `Test_Receiver_MethodName` otherwise it gets hard to read when mixing multiple-words together. In example `NetworkService` with private method `getPeer` would result in `TestNetworkServicegetPeer` which is worst than `Test_NetworkService_getPeer`. But don't get me wrong, I dislike snake_case apart from this niche usage haha